libraries/redcore/joomla/image/image.php

```diff
@@ -697,14 +697,10 @@ public function resize($width, $height, $createNew = true, $scaleMethod = self::
 			// Set the transparent color values for the new image.
 			imagecolortransparent($handle, $color);
 			imagefill($handle, 0, 0, $color);
-
-			imagecopyresized($handle, $this->handle, $offset->x, $offset->y, 0, 0, $dimensions->width, $dimensions->height, $this->getWidth(), $this->getHeight());
-		}
-		else
-		{
-			imagecopyresampled($handle, $this->handle, $offset->x, $offset->y, 0, 0, $dimensions->width, $dimensions->height, $this->getWidth(), $this->getHeight());
 		}
 
+		imagecopyresampled($handle, $this->handle, $offset->x, $offset->y, 0, 0, $dimensions->width, $dimensions->height, $this->getWidth(), $this->getHeight());
```

Comment:
Because for resize function, the result of imagecopyresized give pixelated result, only result from imagecopyresampled return good quality thumbnail.